.circleci/config.yml

```diff
@@ -101,14 +99,18 @@ jobs:
     <<: *job_defaults
     steps:
       - checkout
+      - run:
+          <<: *install_dependencies
       - run: date +%F > date
       - restore_cache:
           <<: *restore_cache
-      - run: sudo pip install pyyaml
+      - run: sudo pip install webtest
+      - run: sudo pip install configparser
+      - run: sudo pip install coverage
```

Comment:
Do we still need these as well? Shouldn't `install_third_party_libs` take care of them?

Reply:
Yes these are required else there are some errors in CircleCI related to environment. These are also installed in the install_third_party_libs but still they are needed here

Reply:
If every other script can run without it, that's fine with me.